src/applications/personalization/profile-2/components/EmailAddressNotification.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import recordEvent from 'platform/monitoring/record-event';
+
+const EmailAddressNotification = () => (
+  <>
+    <p className="vads-u-margin--0">
+      To update the email address you use to sign in, go to the account where
+      you manage your settings and identity information. Any email updates you
+      make there will automatically update on VA.gov.
+    </p>
+    <p className="vads-u-margin-bottom--0">
+      <a
+        href="https://wallet.id.me/settings"
+        target="_blank"
+        rel="noopener noreferrer"
+        onClick={() => {
+          recordEvent({ event: 'update-sign-in-email-address-clicked' });
```

Comment:
@erikphansen I was not sure what string to log here in `recordEvent`. What is the normal procedure here? Do we come up with descriptive names ourselves, or do we need to contact another team?

Reply:
We should probably add in analytics recording later. We don't currently record these clicks to GA (based on looking at https://github.com/department-of-veterans-affairs/vets-website/blob/ecfd7d44e37e85c76010522d1956521b1b5b221b/src/applications/personalization/account/components/AccountMain.jsx and https://github.com/department-of-veterans-affairs/vets-website/blob/42135a1b4ea1cc8741fed41aa3bd7053da99ef11/src/applications/personalization/account/components/LoginSettings.jsx). But, yes, we need to coordinate with the analytics folks

Reply:
@jonwehausen do you have any insights? 😄